### PR TITLE
Add KDE Plasma Icon

### LINKED ISSUE #11559: Request: KDE Plasma
### Brand Name

KDE Plasma

### Website

https://kde.org/plasma-desktop/

### Popularity Metric

Similarweb Global Rank: [44,632](https://www.similarweb.com/website/kde.org) (KDE Plasma is one of the primary products from KDE)

### Official Resources for Icon and Color

- SVG: https://kde.org/stuff/clipart/logo/plasma-logo-monochrome.svg (from press kit)

### Additional Comments

Press kit available: https://kde.org/stuff/clipart/

### PR DESCRIPTION
**Pull request by kodalegit**
image

**Issue:** closes #11559

**Popularity metric:** Similarweb Global Rank - [46623](https://www.similarweb.com/website/kde.org/#overview)

### Checklist

- [X] I updated the JSON data in `_data/simple-icons.json`
- [X] I optimized the icon with SVGO or SVGOMG
- [X] The SVG `viewbox` is `0 0 24 24`

### Description
I picked the hex value from the most prominent color in the brand's logo. You can find the press kit [here](https://kde.org/stuff/clipart/).

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11785,6 +11785,17 @@
...
+        {
+            "title": "Plasma",
+            "hex": "1D99F3",
+            "source": "https://kde.org/stuff/clipart/",
+            "guidelines": "https://kde.org/stuff/clipart/",
+            "aliases": {
+                "aka": [
+                    "KDE Plasma"
+                ]
+            }
```
Comment (uncenter) [inline _data/simple-icons.json:11797]:
```suggestion
            "title": "KDE Plasma",
            "hex": "1D99F3",
            "source": "https://kde.org/stuff/clipart/",
            "guidelines": "https://kde.org/stuff/clipart/"
```

I think we just call it KDE Plasma.

File: icons/plasma.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Plasma</title><path d="M5.993 0a1.495 1.5 0 1 0 0 3.002 1.496 1.501 0 1 0 0-3.001zm10.505 0-3 3.002 4.5 4.498-4.5 4.5 3 3L21 10.5l3-3ZM2.242 9a2.242 2.25 0 1 0 0 4.5 2.242 2.25 0 1 0 0-4.5zm6.755 9a2.991 3.002 0 0 0-3.004 3 2.99 3.001 0 0 0 3.004 3 2.99 3 0 1 0 0-6z"/></svg>
```
Comment (uncenter) [inline icons/plasma.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>KDE Plasma</title><path d="M5.993 0a1.495 1.5 0 1 0 0 3.002 1.496 1.501 0 1 0 0-3.001zm10.505 0-3 3.002 4.5 4.498-4.5 4.5 3 3L21 10.5l3-3ZM2.242 9a2.242 2.25 0 1 0 0 4.5 2.242 2.25 0 1 0 0-4.5zm6.755 9a2.991 3.002 0 0 0-3.004 3 2.99 3.001 0 0 0 3.004 3 2.99 3 0 1 0 0-6z"/></svg>
```

File: icons/plasma.svg
Comment (uncenter) [inline icons/plasma.svg:1]: Rename file to `icons/kdeplasma.svg`.

File: icons/plasma.svg (hunk 2 above)
Comment (uncenter) [reply icons/plasma.svg:1]: Seeing issues with all 3 circles. Make sure not to over-optimize! 4 should be good here.
CleanShot 2024-09-14 at 20 55 22

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/kdeplasma.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>KDE Plasma</title><path d="M5.993 0a1.495 1.5 0 1 0 0 3.002 1.496 1.501 0 1 0 0-3.001zm10.505 0-3 3.002 4.5 4.498-4.5 4.5 3 3L21 10.5l3-3ZM2.242 9a2.242 2.25 0 1 0 0 4.5 2.242 2.25 0 1 0 0-4.5zm6.755 9a2.991 3.002 0 0 0-3.004 3 2.99 3.001 0 0 0 3.004 3 2.99 3 0 1 0 0-6z"/></svg>
```
Comment (uncenter) [inline icons/kdeplasma.svg:1]: Still seeing gaps. This looks good on my end though:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>KDE Plasma</title><path d="M6 0c-.831 0-1.5.669-1.5 1.5S5.169 3 6 3s1.5-.669 1.5-1.5S6.831 0 6 0m10.5 0-3 3L18 7.5 13.5 12l3 3 4.5-4.5 3-3zM2.25 9A2.245 2.245 0 0 0 0 11.25a2.245 2.245 0 0 0 2.25 2.25 2.245 2.245 0 0 0 2.25-2.25A2.245 2.245 0 0 0 2.25 9M9 18c-1.662 0-3 1.338-3 3s1.338 3 3 3 3-1.338 3-3-1.338-3-3-3"/></svg>
```

**Review by PratyushJayachandran — APPROVED** [review]
Thank you @kodalegit for contributing. Thanks to @runxel and @uncenter for resolving the issues🎉

**Pull request merged**